it's normal that the acl_* symbols are not found if the application
is not linked against -lacl. these errors harmless, but mighty
annoying.
---
diff --git libfakeroot.c libfakeroot.c
index 3e80e38..d3b9232 100644
--- libfakeroot.c
+++ libfakeroot.c
@@ -263,7 +263,7 @@ void load_library_symbols(void){
     *(next_wrap[i].doit)=dlsym(get_libc(), next_wrap[i].name);
 #ifdef LIBFAKEROOT_DEBUGGING
     /* illumos libc creates noise if symbols is not found (e. g. acl_get())*/
-    if ( (msg = dlerror()) != NULL){
+    if ( fakeroot_debug && (msg = dlerror()) != NULL) {
       fprintf (stderr, "dlsym(%s): %s\n", next_wrap[i].name, msg);
     }
 #endif /* LIBFAKEROOT_DEBUGGING */
